tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -70,6 +70,9 @@ func (d *deployer) Up() error {
 			if err := d.aws.EnsureS3Bucket(ctx, d.stateStore(), false); err != nil {
 				return err
 			}
+			if err := d.aws.EnsureS3Bucket(ctx, d.discoveryStore(), false); err != nil {
```

Comment:
I think we want to make them public ? cc @rifelpet 

```suggestion
			if err := d.aws.EnsureS3Bucket(ctx, d.discoveryStore(), true); err != nil {
```

Reply:
The existing bucket (`s3://k8s-kops-ci-prow`) is public, but I'm not sure if that's because it's required by tests or because we want boskos accounts to be able to read from this bucket. Also, do we need to migrate existing data from there?

Reply:
I confirm, data is irrelevant. If this is the discovery bucket, we need it to be public, or put CloudFront in front of it.